src/app/dataset-flow/dataset-flow-details/tabs/flow-details-history-tab/flow-details-history-tab.component.html

```diff
@@ -45,4 +45,11 @@
             </div>
         </li>
     </ul>
+
+    <ng-container *ngIf="isAdmin && grafanaFlowHistoryUrl">
+        <p class="position-relative my-4 box p-4">
+            You can <a href="#">see history in Grafana</a>
+            <mat-icon class="admin-view text-muted"> admin_panel_settings</mat-icon>
+        </p>
+    </ng-container>
```

Comment:
Don't add this yet, we have no URL for flow history

Reply:
Okey, removed